cli/internal/packager/common.go

```diff
@@ -187,7 +204,7 @@ func HandleIfURL(packagePath string, shasum string, insecureDeploy bool) string
 func isValidFileExtension(filename string) bool {
 	for _, extension := range config.GetValidPackageExtensions() {
 		if strings.HasSuffix(filename, extension) {
-			logrus.WithField("packagePath", filename).Warn("Package extension is valid.")
+			message.Warnf("Extension for %s is invalid", filename)
```

Comment:
I'm not sure why this was a warning before but this looks like we have our log flipped.

Reply:
hmm that's super odd......yeah that whole thing doesn't even warrant a log at that level I think